server/src/main/java/com/google/musicanalysis/site/YoutubeServlet.java

```diff
@@ -118,14 +132,23 @@ protected void doGet(HttpServletRequest req, HttpServletResponse res)
             return;
         }
 
-        String youtubeResBody = getYoutubeRes(API_KEY, accessToken.toString());
-        int numVideos = Integer.parseInt(req.getParameter("num_videos"));
+        String numVideosParam = req.getParameter("num_videos");
+        int numVideos = Integer.parseInt(DEFAULT_NUM_VIDS);
```

Comment:
DEFAULT_NUM_VIDS should just be defined as an integer so you don't have to convert it, it doesn't look like it's used anywhere that requires it to be a string.